plugin.video.themoviedb.helper/resources/lib/api/kodi/rpc.py

```diff
@@ -176,15 +199,22 @@ def _get_database(self, dbtype, tvshowid=None, attempt_reconnect=False, logging=
         return self.get_database(dbtype, tvshowid, attempt_reconnect)
 
     def get_database(self, dbtype, tvshowid=None, attempt_reconnect=False, logging=True):
+        from xbmcvfs import Stat
+        cache_name = f'db.{dbtype}.{tvshowid}'
+        cache_data = self._cache.get_cache(cache_name)
+        db_updated = Stat('special://database/MyVideos119.db').st_mtime() or -1
```

Comment:
Looks like an hack to me. Is it really needed to stat our db files?
This is usually not allowed...

Reply:
It is definitely a hack. I build a dictionary based upon the database and then cache it for quick referencing against item IDs, so I only want to rebuild that dictionary if the library database changes. I couldn't find a way to get a "library last updated" type of value, hence the Stat. 

Happy to replace it with something else if there's a better option for checking when the library was updated last.

Reply:
Decided to remove the Stat and separate caching entirely. It only really has a speed advantage when the user has many thousands of items in their library, at which point their usage pattern is probably pretty questionable anyway...